core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -845,9 +848,7 @@ var ExplorationEditorMainTab = function() {
     await answerDescription.click();
     var ruleDropdownElement = await element.all(by.cssContainingText(
       '.select2-results__option', ruleDescriptionInDropdown)).first();
-    await waitFor.visibilityOf(
-      ruleDropdownElement, 'Rule dropdown element takes too long to appear');
-    await ruleDropdownElement.click();
+    await action.click('Rule Dropdown Element', ruleDropdownElement);
```

Comment:
I think we have a dedicated action for select2 dropdowns, right?

Reply:
I tried to use `action.select2()`, but I wasn't quite able to make it work. Do you have any ideas?

Reply:
It looks like the `action.select2()` function first finds an element with class `.select2-results`, and then looks for a `<li>` item within it. Does the dropdown here match that format?